mongo/options/runcmdoptions.go

```diff
@@ -8,23 +8,25 @@ package options
 
 import "go.mongodb.org/mongo-driver/mongo/readpref"
 
-// RunCmdOptions represents all possible options for a runCommand operation.
+// RunCmdOptions represents options that can be used to configure a RunCommand operation.
 type RunCmdOptions struct {
-	ReadPreference *readpref.ReadPref // The read preference for the operation.
+	// The read preference to use for the operation. The default value is nil, which means that the read preference
```

Comment:
I thought RunCommand by default routed to the primary.

Reply:
See https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#use-of-read-preferences-with-commands

Reply:
Looks like we are correctly using primary as the default. Updated the comment.